src/participant_network.star

```diff
@@ -98,7 +102,7 @@ def launch_participant_network(
                 run="mkdir -p /shadowfork && \
                     curl -o /shadowfork/latest_block.json https://ethpandaops-ethereum-node-snapshots.ams3.digitaloceanspaces.com/"
                 + base_network
-                + "/geth/latest/latest_snapshot_block.json",
+                + "/geth/" + shadowfork_block + "/_snapshot_eth_getBlockByNumber.json",
```

Comment:
this is not what Rafael called the json files. Maybe we can rename this to be just block.json?